README.md

```diff
@@ -1 +1,50 @@
-# styleguide
+# Ombulabs::StyleGuide
```

Comment:
@FionaDL It seems that you missed my comment here.

Reply:
I did miss changing the OmbuLabs. Sorry!